src/main/java/marquez/db/DbTableVersionDao.java

```diff
@@ -0,0 +1,61 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package marquez.db;
+
+import java.util.List;
+import java.util.Optional;
+import java.util.UUID;
+import marquez.db.mappers.DbTableVersionRowMapper;
+import marquez.db.models.DbTableVersionRow;
+import org.jdbi.v3.core.Handle;
+import org.jdbi.v3.sqlobject.SqlObject;
+import org.jdbi.v3.sqlobject.config.RegisterRowMapper;
+import org.jdbi.v3.sqlobject.statement.SqlQuery;
+
+@RegisterRowMapper(DbTableVersionRowMapper.class)
+public interface DbTableVersionDao extends SqlObject {
+  default void insert(DbTableVersionRow row) {
```

Comment:
I assume this insert function lives in a transaction context. correct ?

Reply:
Yep! It definitely should 👍